app/code/Magento/MediaGallery/Model/Directory/Config.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGallery\Model\Directory;
+
+use Magento\Framework\Config\DataInterface;
+
+/**
+ * Media gallery directory config
+ */
+class Config
+{
+    private const XML_PATH_BLACKLIST_PATTERNS = 'blacklist/patterns';
```

Comment:
Create interface getBlacklistPatternsInterface

Reply:
remove get

Reply:
Why?

Reply:
get is removed and interface was created, but interface has to be moved to MediaGalleryApi\Model\BlacklistPatternsConfigInterface::get